bitnami/mariadb/values.yaml

```diff
@@ -1129,6 +1129,31 @@ metrics:
     ## @param metrics.serviceMonitor.labels Extra labels for the ServiceMonitor
     ##
     labels: {}
+  ## Prometheus Operator PrometheusRule configuration
+  ##
+  prometheusRules:
+    ## @param metrics.prometheusRules.enabled if `true`, creates a Prometheus Operator PrometheusRule (also requires `metrics.enabled` to be `true`, and makes little sense without ServiceMonitor)
+    ##
+    enabled: false
+    ## @param metrics.prometheusRules.selector [object] Additional labels to add to the PrometheusRule so it is picked up by the operator
+    ## If using the [Helm Chart](https://github.com/helm/charts/tree/master/stable/prometheus-operator) this is the name of the Helm release and 'app: prometheus-operator'
+    ##
+    selector:
+      app: prometheus-operator
+      release: prometheus
+    ## @param metrics.prometheusRules.rules PrometheusRule rules to configure
+    ## e.g:
+    ##  - alert: MariaDB-Down
+    ##    annotations:
+    ##      message: 'MariaDB instance {{ $labels.instance }} is down'
+    ##      summary: MariaDB instance is down
+    ##    expr: absent(up{job="mariadb"} == 1)
+    ##    labels:
+    ##      severity: warning
+    ##      service: mariadb
+    ##    for: 5m
+    ##
+    rules: {}
```

Comment:
Are you using the [readme-generator tool](https://github.com/bitnami-labs/readme-generator-for-helm) to autogenerate the README.md tables based on this?

Reply:
Could you follow the same format we use in other charts (e.g. Apache) for this kind of objects? See https://github.com/bitnami/charts/blob/master/bitnami/apache/values.yaml#L517

Reply:
Yes, I do

Reply:
Sure

Reply:
I ran this:
```
readme-generator -r README.md -v values.yaml
```
and this is the output:
```
Skipping check for primary.configuration
Skipping check for secondary.configuration
Skipping check for metrics.annotations.prometheus.io/scrape
Skipping check for metrics.annotations.prometheus.io/port
Skipping check for metrics.extraArgs.primary
Skipping check for metrics.extraArgs.secondary
Skipping check for metrics.prometheusRules.selector.app
Skipping check for metrics.prometheusRules.selector.release
Skipping check for networkPolicy.metrics.podSelector
Skipping check for networkPolicy.metrics.namespaceSelector
Skipping check for networkPolicy.ingressRules.primaryAccessOnlyFrom.namespaceSelector
Skipping check for networkPolicy.ingressRules.primaryAccessOnlyFrom.podSelector
Skipping check for networkPolicy.ingressRules.primaryAccessOnlyFrom.customRules
Skipping check for networkPolicy.ingressRules.secondaryAccessOnlyFrom.namespaceSelector
Skipping check for networkPolicy.ingressRules.secondaryAccessOnlyFrom.podSelector
Skipping check for networkPolicy.ingressRules.secondaryAccessOnlyFrom.customRules
Skipping check for networkPolicy.egressRules.customRules
INFO: Checking missing metadata...
INFO: Metadata is correct!
INFO: Found parameters section at: ## Parameters
INFO: Found section end at: The above parameters map to the env variables defined in [bitnami/mariadb](https://github.com/bitnami/bitnami-docker-mariadb). For more information please refer to the [bitnami/mariadb](https://github.com/bitnami/bitnami-docker-mariadb) image documentation.
INFO: Inserting the new table into the README...
```

Reply:
And the README.md is already is last commit

Reply:
> Could you follow the same format we use in other charts (e.g. Apache) for this kind of objects? See https://github.com/bitnami/charts/blob/master/bitnami/apache/values.yaml#L517

I understood now.
I'll make the changes.